src/components/Content.tsx

```diff
@@ -164,15 +164,36 @@ const Content = ({...props}: ContentProps) => {
     }
     
     const fetchLastFormDefinition = async () => {
-        let currentForm = await helper.callGraphQL<FormDefinitionByCreatedAt>(customQueries.formByCreatedAtt, customQueries.formByCreatedAtInputConsts);
-        if (currentForm.data) {
-            let formDef = currentForm.data.formByCreatedAt.items[0];
-            // console.log("FormDef:", formDef);
-            setFormDefinition(formDef);
-            let quAns = createQuestionAnswers(formDef);
-            let userAnswers = await getUserAnswers();
-            setFirstAnswers(quAns, userAnswers);
-            // console.log("Completed questuions");
+        let nextToken: string | null = null;
+        let questions: Question[] = [];
+        let formDefPaginated: FormDefinitionPaginated = undefined;
+        do {
+            let currentForm: any = await helper.callGraphQL<FormDefinitionByCreatedAtPaginated>(customQueries.formByCreatedAtPaginated, {...customQueries.formByCreatedAtInputConsts, nextToken: nextToken});
```

Comment:
Should any be  changed to a type, to remove annoying any types in typescript
```suggestion
            let currentForm = await helper.callGraphQL<FormDefinitionByCreatedAtPaginated>(customQueries.formByCreatedAtPaginated, {...customQueries.formByCreatedAtInputConsts, nextToken: nextToken});
```